content/en/docs/v3.6/op-guide/security.md

```diff
@@ -239,6 +239,98 @@ To disable certificate chain checking, invoke curl with the `-k` flag:
 $ curl -k https://127.0.0.1:2379/v2/keys/foo -Xput -d value=bar -v
 ```
 
+## Example 5: Convert an existing non-TLS etcd cluster to TLS
+
+Previously, we described that etcd supports two types of communication: **client-to-server** and **server-to-server (peer communication)**. In this section, we will prepare configuration steps to migrate both communication types to TLS in your existing etcd cluster.
+
+> **Important**: Before restarting any etcd members, you must first add the TLS certificates and configuration to each etcd node.
+
+---
+
+### 1. Check current cluster status
+
+```sh
+export ETCDCTL_API=3
+
+# Define etcd node IPs
+HOST_1=ip_address_etcd_node_1
+HOST_2=ip_address_etcd_node_2
+HOST_3=ip_address_etcd_node_3
+
+# Get member list
+etcdctl --endpoints=${HOST_1}:2379,${HOST_2}:2379,${HOST_3}:2379 member list
+```
+
+You will get an output similar to:
+
+```sh
+10f4390f4a904a13, started, etcd-node-1, http://ip_address_etcd_node_1:2380, http://ip_address_etcd_node_1:2379, false
+18e03f3590bf045c, started, etcd-node-2, http://ip_address_etcd_node_2:2380, http://ip_address_etcd_node_2:2379, false
+ca7a7d6fe0984fd9, started, etcd-node-3, http://ip_address_etcd_node_3:2380, http://ip_address_etcd_node_3:2379, false
+```
+
+> Save the **member IDs** and corresponding **IP addresses** — you'll need them to update peer URLs.
+
+---
+
+### 2. Enable TLS for server-to-server communication
+
+Update each etcd member’s peer URL to use `https://`:
+
+```sh
+etcdctl member update 10f4390f4a904a13 --peer-urls=https://ip_address_etcd_node_1:2380
+etcdctl member update 18e03f3590bf045c --peer-urls=https://ip_address_etcd_node_2:2380
+etcdctl member update ca7a7d6fe0984fd9 --peer-urls=https://ip_address_etcd_node_3:2380
+```
+
+---
+
+### 3. Enable TLS for client-to-server communication
+
+Edit the systemd unit file (`/etc/systemd/system/etcd.service`) or drop-in override config on each node to add the following flags:
```

Comment:
minor comment: suggest not to mention the `/etc/systemd/system/etcd.service`, we just need to clarify which etcd options need to be updated.

Reply:
I'll update it